frontend/src/index.tsx

```diff
@@ -19,30 +19,40 @@ import { BrowserRouter as Router } from 'react-router-dom';
 import EmptyLayout from 'layouts/EmptyLayout';
 import LoginLoading from 'features/account/LoginLoading';
 import { store } from 'store/store';
-import { TenantProvider } from 'tenants';
+import { TenantConsumer, TenantProvider } from 'tenants';
+import { ThemeProvider } from 'styled-components';
+
+// eslint-disable-next-line import/no-webpack-loader-syntax
+const css = require('sass-extract-loader?{"plugins": ["sass-extract-js"]}!./_variables.scss');
```

Comment:
I would love a better way to do this. 
This extracts scss variables and provides a way to include them into the styled component theme.